src/main/java/org/aion/wallet/ui/components/ReceiveController.java

```diff
@@ -41,6 +47,13 @@ private void handleAccountChanged(final AccountEvent event) {
         if (AccountEvent.Type.CHANGED.equals(event.getType())) {
```

Comment:
this should be `EnumSet.of(AccountEvent.Type.CHANGED, AccountEvent.Type.ADDED).contains(event.getType())` - see `HeaderPaneControls` for a model

And also, there should be a check to refresh only if the account is active